map_server/src/main.cpp

```diff
@@ -78,7 +79,7 @@ class MapServer
         //std::ifstream fin((fname + ".yaml").c_str());
         std::ifstream fin(fname.c_str());
         if (fin.fail()) {
-          fprintf(stderr, "Map_server could not open %s.\n", fname.c_str());
+          RCUTILS_LOG_ERROR("Map_server could not open %s.\n", fname.c_str());
```

Comment:
> Please remove the trailing \n too. 

Good catch! 

At this point (as the change had been made earlier), I feel it is best to just go ahead and change to `RCUTILS_LOG_*`.